web_client/src/store/index.ts

```diff
@@ -49,9 +50,9 @@ function prepareProxyManager(proxyManager) {
       view.setOrientationAxesVisibility(false);
       view.getRepresentations().forEach((representation) => {
         representation.setInterpolationType(InterpolationType.NEAREST);
-        representation.onModified(() => {
+        representation.onModified(macro.debounce(() => {
           view.render(true);
-        });
+        }, 0));
```

Comment:
I think having a debounce with a 0 time parameter merits a comment explaining it.

Reply:
Does the comment in [b5444a7](https://github.com/OpenImaging/miqa/pull/633/commits/b5444a7cce695b048450122c2fde71fe8862021f) suffice? Or should I explain further about the simultaneous changes?

Reply:
LGTM